include/ultra64/gu.h

```diff
@@ -1,8 +1,51 @@
 #ifndef ULTRA64_GU_H
 #define ULTRA64_GU_H
 
+#include "ultratypes.h"
+#include "mbi.h"
+
 #define GU_PI 3.1415926
 
 #define ROUND(x) (s32)(((x) >= 0.0) ? ((x) + 0.5) : ((x) - 0.5))
 
+void guMtxIdent(Mtx*);
```

Comment:
Relating to the arg question again, some functions here have arg names and some don't. Any particular reason why?